services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/service.py

```diff
@@ -158,6 +165,7 @@ def from_scheduler_data(
         )
 
     class Config(ServiceDetails.Config):
+        keep_untouched = (cached_property,)
```

Comment:
does that mean it cannot be changed afterwards? what is the difference with the frozen attribute? just set on one specific attribute?

Reply:
This is necessary in oder to use `cached_property` with Pydantic models, otherwise it raises a strange error regarding threading when parsing the model.